lib/facility.rb

```diff
@@ -1,14 +1,189 @@
+require 'pry'
+require 'date'
+require 'faraday'
+require 'json'
+require './lib/dmv'
+require './lib/vehicle'
+require './lib/facility'
+require './lib/registrant'
+require './lib/vehicle_factory'
+require './lib/dmv_data_service'
+
 class Facility
-  attr_reader :name, :address, :phone, :services
+  attr_reader :name,
+              :address,
+              :phone,
+              :services,
+              :registered_vehicles,
+              :collected_fees
+
+  attr_accessor :name, :address, :phone, :services
 
-  def initialize(name, address, phone)
-    @name = name
-    @address = address
-    @phone = phone
+  def initialize(hash)
+    @name = hash[:name]
+    @address = hash[:address]
+    @phone = hash[:phone]
     @services = []
+    @registered_vehicles = []
+    @collected_fees = 0
   end
 
-  def add_services(service)
+  def add_service(service)
     @services << service
   end
+  #refactor
+  def register_vehicle(car)
+    if services.find {|service| service.include?('Vehicle Registration')}
```

Comment:
you could probably slim this line down a bit by doing something like: 

`if @services.include?("Vehicle Registration")`